### PR TITLE
Form input settings

### PR DESCRIPTION
**Pull request by natewiebe13**
Goal:
- Add the ability to set the padding for form elements.
- Have consistent font-weight for inputs and selects

Notes: The original height calculation was using a hardcoded 1.5 and was being multiplied to the form spacing. I'm guessing this was to work like the line height and was being multiplied to the wrong variable. Let me know if it had a different purpose.

### CODE REVIEW

**Comment by kball** [comment]
@andycochran @brettsmason @ncoden what do y'all think?

**Comment by rafibomb** [comment]
The most cross-browser way to increase the size of the inputs is with height and width.

Padding is an issue on FF
screen shot 2017-02-02 at 10 00 17 am

**Comment by natewiebe13** [comment]
@rafibomb, as long as the calculation for height is correct, it shouldn't hurt to update the padding to what would make sense. I don't seem to be having the issue you are. Can you provide a link?

**Comment by rafibomb** [comment]
@natewiebe13 I opened the Foundation docs in FF http://foundation.zurb.com/sites/docs/forms.html

And in the inspector increased the input padding. It doesn't keep the placeholder in the middle.

**Comment by natewiebe13** [comment]
@rafibomb correct, because the height has to be adjusted as well. My PR takes care of that as well. Without touching the inspector, do you notice any issues when adjusting the setting?

**Comment by kball** [comment]
@andycochran can you take a look at this PR?

File: scss/settings/_settings.scss [hunk 1]
```diff
@@ -375,11 +375,13 @@ $input-placeholder-color: $medium-gray;
 $input-font-family: inherit;
 $input-font-size: rem-calc(16);
 $input-font-weight: $global-weight-normal;
+$input-line-height: $global-lineheight;
```
Comment (IamManchanda) [inline scss/settings/_settings.scss]: Setting file shouldn't be added .... Automatically generates!

**Comment by IamManchanda** [comment]
LGTM ..... cc @kball @rafibomb for 6.4!

**Comment by rafibomb** [comment]
It does work using the settings file, nice PR!

**Comment by rafibomb** [comment]
Migration notes: 
new variables
- `$input-line-height: $global-lineheight;`
- `$input-padding: $form-spacing / 2;`

**Comment by kball** [comment]
I think this will actually "just work" without a migration, but adding the new variable will give you the control... @rafibomb was there a particular reason you wanted a note on the new settings?

**Pull request merged**